Update getting help with altis guide for new dashboard (#257)
humanmade/altis-documentation: other-docs/guides/getting-help-with-altis.md

## Create Support Ticket

### Selecting a support issue type

To help guide you to finding the right kind of help, there's four categories a support ticket can fall under. They all have their own labels, but we'll go over them quickly.

**Report an Incident**

Incidents are unplanned events that negatively impact service, or the quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.

From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
If the site is down for example, select 'Report an Incident'.

**Report an ongoing issue**

Issues, also known as 'Problems' are the cause of, or can be the potential cause of future Incidents. If you're encountering a problem which could cause potential Incidents, or if you've run into a bug, select 'Report an ongoing issue'.

**Request an infrastucture change**

> There’s an array of tasks that you might need help with. Typically, this might be the provisioning of an additional environemnent, or it could be to run an operation that requires a cloud engineer, upgrading PHP for example.

Reviewer: ```suggestion
There’s an array of tasks that you might need help with. Typically, this might be the provisioning of an additional environment, or it could be an operation that requires a cloud engineer to run, such as upgrading the PHP version.
```

> Tasks should be well described with detailed requirements and time scales. We cannot guarantee will not be able to service same-day requests, if there's capacity to do so, we will.

Reviewer: ```suggestion
Tasks should be well described with detailed requirements and time scales. We cannot guarantee we will be able to service same-day requests, however if there is capacity to do so, we will.
```

**Ask a question**

Have a question for us? Ask!

If you have a technical question or require clarification on a feature, just open a support question and our support team will do their best to answer you.

### Filling the form

**Application**

> Select the specific Application or select all Application(s)/Environment(s) to which the support ticket concerns.

Reviewer: ```suggestion
Select the specific Application or select all Application(s)/Environment(s) that the support ticket concerns.
```

**Priority**

> You can change the priority of a support issue if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority.

Reviewer: Should we describe the intended use of priorities or no? e.g. urgent should only be used for site outage or serious problem happening right now. If you have other support requests in the queue then you could set questions or newer requests to a lower priority for example.

Something along those lines? Not a requirement though.

Author: That’s a good idea actually. I hadn’t thought of it like that. Commit incoming!

Author: I took a bash at guiding use for priorities.

> You can change the priority of a support issue if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority. For example, the 'Urgent' priority should only be used if your experience a service outage. The other priorities; low to high, can be used to indicate which of your own open tickets you'd like to be prioritised.

Reviewer: Maybe to simplify it a bit:
```suggestion
You can change the priority of a support issue if you need to. In most cases you shouldn't need to change this, but feel free to let us know if something is a priority. For example, the 'Urgent' priority should only be used if you are experiencing a service outage. The other priorities; low to high, can be used to indicate the order you would like us to tackle your open tickets in.
```

**Subject**

Short description of the support request.

**Description**

The body of the support request.

**CCs**

> You can CC additional relevant parties to the support request if desired. They will recieve carbon copies of the correspondance.

Reviewer: ```suggestion
You can CC additional relevant parties to the support request if desired. They will receive carbon copies of the correspondence.
```
